src/App.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './App.scss';
+import { List } from './components/List';
+
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+class App extends React.Component {
+  state = {
+    selectedProducts: [],
+    displayProducts: '',
+    isActive: false,
+  }
+
+  addProducts = (product) => {
+    const {selectedProducts} = this.state;
+    const index = selectedProducts.indexOf(product);
+
+    if (selectedProducts.includes(product)) {
+      selectedProducts.splice(index, 1);
+
+      return this.setState({
+        displayProducts: selectedProducts.join(', '),
+      });
+    }
+
+    if (!selectedProducts.includes(product)) {
+      selectedProducts.push(product);
+
+      return this.setState({
+        displayProducts: selectedProducts.join(', '),
+      });
```

Comment:
```suggestion
      this.setState({
        displayProducts: selectedProducts.join(', '),
      });
```
можно без `return `прописать

Reply:
можно, но тогда след проверка окажется true, потому что элемент уже отображается в заголовке. Мне нужно выполнить проверку и выйти из функции сразу.

Reply:
Да, точно, ты здесь работаешь со значением `selectedProducts`. Упустил этот момент. Я для таких целей создавал копию `selectedProducts` и работал с ней.